### PR TITLE
[hi] added content/hi/docs/concepts/_index.md

### PR DESCRIPTION
**Pull request by akash-kumar-saw**
**PR Number : #39568**

added content/hi/docs/concepts/_index.md

### CODE REVIEW

**Review by vaibhav2107 — COMMENTED** [review]
Hi @akash-kumar-saw Thanks for the PR
I think the last line will add more meaning if it can be merged with the previous one like 
.... प्रतिनिधित्व करने के लिए उपयोग करता है,,और कुबेरनेट्स कैसे काम करता है, इसकी गहरी समझ प्राप्त करने में आपकी मदद करता है।

File: content/hi/docs/concepts/_index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+title: अवधारणाओं
+main_menu: true
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+अवधारणा अनुभाग आपको कुबेरनेट्स प्रणाली के हिस्सों के बारे में जानने में मदद करता है और कुबेरनेट्स आपके {{< glossary_tooltip text="cluster" term_id="cluster" length="all" >}} का प्रतिनिधित्व करने के लिए उपयोग करता है, और आपको एक गहन जानकारी प्राप्त करने में मदद करता है। कुबेरनेट्स कैसे काम करता है इसकी समझ।
```
Comment (amolmote) [inline content/hi/docs/concepts/_index.md:10]:
```suggestion
अवधारणा अनुभाग आपको कुबेरनेट्स प्रणाली के हिस्सों के बारे में जानने में मदद करता है और कुबेरनेट्स आपके {{< glossary_tooltip text="क्लस्टर" term_id="cluster" length="all" >}} का प्रतिनिधित्व करने के लिए उपयोग करता है, और आपको  कुबेरनेट्स कैसे काम करता है इसकी एक गहन जानकारी प्राप्त करने में मदद करता है। 
```

Comment (amolmote) [reply content/hi/docs/concepts/_index.md:10]: cc @divya-mohan0209 & @bishal7679  your thoughts on this change?

**Review by amolmote — CHANGES_REQUESTED** [review]

**Comment by akash-kumar-saw** [comment]
Hi @vaibhav2107 & @amolmote,
I have commited your suggestions,
You can review it now and can approve it :)

File: content/hi/docs/concepts/_index.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+---
+title: अवधारणाओं
+main_menu: true
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+अवधारणा अनुभाग आपको कुबेरनेट्स प्रणाली के हिस्सों के बारे में जानने में मदद करता है और कुबेरनेट्स आपके {{< glossary_tooltip text="क्लस्टर" term_id="cluster" length="all" >}} का प्रतिनिधित्व करने के लिए उपयोग करता है, और आपको कुबेरनेट्स कैसे काम करता है, इसकी गहरी समझ प्राप्त करने में आपकी मदद करता है।
```
Comment (Babapool) [inline content/hi/docs/concepts/_index.md:10]:
```suggestion
अवधारणा अनुभाग आपको कुबेरनेट्स प्रणाली के हिस्सों के बारे में जानने में मदद करता है जिसका उपयोग कुबेरनेट्स आपके {{< glossary_tooltip text="क्लस्टर" term_id="cluster" length="all" >}} का प्रतिनिधित्व करने के लिए करता है, और कुबेरनेट्स कैसे काम करता है, इसकी गहरी समझ प्राप्त करने में आपकी मदद करता है।
```

**Review by Babapool — COMMENTED** [review]
Some suggestions from my side, hope you find them useful. Thanks.

File: content/hi/docs/concepts/_index.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
+---
+title: अवधारणाएँ
+main_menu: true
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+अवधारणा अनुभाग आपको कुबेरनेट्स प्रणाली के हिस्सों के बारे में जानने में मदद करता है जिसका उपयोग कुबेरनेट्स आपके {{< glossary_tooltip text="क्लस्टर" term_id="cluster" length="all" >}} का प्रतिनिधित्व करने के लिए करता है, और कुबेरनेट्स कैसे काम करता है, इसकी गहरी समझ प्राप्त करने में आपकी मदद करता है।
```
Comment (anubha-v-ardhan) [inline content/hi/docs/concepts/_index.md:10]:
```suggestion
अवधारणा अनुभाग आपको कुबेरनेट्स प्रणाली के हिस्सों के बारे में जानने में मदद करता है जिसका उपयोग कुबेरनेट्स आपके {{< glossary_tooltip text="क्लस्टर" term_id="cluster" length="all" >}} का प्रतिनिधित्व करने के लिए करता है, और कुबेरनेट्स कार्यप्रणाली की गहरी समझ प्राप्त करने में आपकी मदद करता है।
```

Comment (anubha-v-ardhan) [reply content/hi/docs/concepts/_index.md:10]: Sounds better to me, wdyt?

**Review by anubha-v-ardhan — CHANGES_REQUESTED** [review]

File: content/hi/docs/concepts/_index.md (hunk 3 above)
Comment (Babapool) [reply content/hi/docs/concepts/_index.md:10]: This sounds much better and captures the meaning of what the docs wanted to convey.

**Comment by Babapool** [comment]
@akash-kumar-saw Can you push an empty commit to (re)trigger the checks once.

**Comment by akash-kumar-saw** [comment]
Hii @anubha-v-ardhan,

I have commited to your suggestions.
You can review it now and can approve it.

Thanks for your precious time : )

**Review by anubha-v-ardhan — COMMENTED** [review]
/lgtm
/approve

Thank you @akash-kumar-saw!

**Pull request merged**